packages/components/src/Switch/Switch.css

```diff
@@ -23,7 +27,16 @@
   appearance: none;
 }
 
-.track:focus {
+:global(.jobber-retheme) .track {
+  width: var(--space-largest);
+  height: var(--space-large);
+  border-color: var(--color-interactive);
```

Comment:
As per Slack, this interactive border is only for hover. Should be grey when unchecked